src/components/UniversalNavbarExpanded/SampleContent.js

```diff
@@ -24,6 +24,10 @@ const playIcon = () => (
  * TODO when Kustomer implementation is ready and we want to include a talk button
```

Comment:
the Kustomer implementation has come and gone :|

Reply:
Ha, good catch, will remove this.